hicommonwealth/commonwealth
# 7023-pwaPromptAnimation

## Link to Issue
Closes: #7023 

## Description of Changes
- Using the `useAnimation` hook, added the animation to both `IOSPrompt` and `AndroidPrompt`

## Test Plan
- Run the website locally
- Ensure the animation occurs for both prompts
